app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,31 @@
+# app/policies/follow_request_policy.rb
+
+class FollowRequestPolicy < ApplicationPolicy
+  # attr_reader :user, :follow_request
+
+  # def initialize(user, follow_request)
```

Comment:
if you are going to inherit "record" from application policy, remove unnecessary/commented out code